src/mol-plugin/behavior/dynamic/camera.ts

```diff
@@ -28,6 +45,8 @@ const DefaultFocusLociBindings = {
         Trigger(B.Flag.Secondary, M.create()),
         Trigger(B.Flag.Primary, M.create({ control: true }))
     ], 'Camera center and focus', 'Click element using ${triggers}'),
+    clickResetCameraOnEmpty: undefined,
```

Comment:
I would prefer `Binding.Empty` instead of making them optional

Reply:
Hi Alexander, I originally had `Binding.Empty` but it was discussed [here](https://github.com/molstar/molstar/pull/742#discussion_r1132139975) that it results in an API change meaning it would be a breaking update. I can revert to what I had originally if needed.

Reply:
What I've meant that is that the code that refrences this property should fall back to the default value, e.g.

```ts
clickResetCameraOnEmpty: Binding.Empty`,
...

const clickResetBinding = params.clickResetCameraOnEmpty ?? DefaultValueForClickResetCameraOnEmpty;
```

The params for behaviors are saved in the snapshots, so this might have caused the binding being `undefined` for older states and raise an exception.